package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "CheckSoldiersHealth",
+  "version": "1.0.0",
+  "description": "",
+  "main": "app.test.js",
+  "scripts": {
+    "test": "jest --collectCoverage"
+},
+"author": "",
+"license": "ISC",
+"dependencies": {
+    "@types/geojson": "^7946.0.10",
```

Comment:
Remove all the dependencies you aren't using. When you start using them add them, not before